src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,24 @@
-export const TodoInfo = () => {};
+import React from 'react';
+import usersFromServer from '../../api/users';
+import { Todo } from '../../types/todo';
+import { UserInfo } from '../UserInfo';
+
+type TodoInfoProps = {
+  todo: Todo;
+};
+
+export const TodoInfo: React.FC<TodoInfoProps> = ({ todo }) => {
+  const currentUser =
+    usersFromServer.find(user => user.id === todo.userId) || null;
```

Comment:
[CODE STYLE] - This logic should be moved to the root component. You should pass the user object to this component as a prop. This will make your component more reusable and easier to test.